lochness/transfer/__init__.py

```diff
@@ -310,7 +310,7 @@ def lochness_to_lochness_transfer_s3(Lochness, general_only: bool = True):
 
     command = f'aws s3 sync \
             {source_directory}/ \
-            s3://{s3_bucket_name}/{s3_phoenix_root}'
+            s3://{s3_bucket_name}/{s3_phoenix_root} --delete'
```

Comment:
Is the `--delete` for delete from AWS after downloading?

Reply:
`--delete` option is for deleting any files on AWS that does not exist in the source PHOENIX on Pronet aggregate server. I'm using `--delete` option, because there are bunch of files created from testing, which confuses our PHOENIX structure on AWS. As discussed in our last meeting, this extra option will be removed in the production server, to keep every files safely on the AWS. I'd like to keep `--delete` for a few more testing rounds.